bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/retriever/AbstractCategoryRetriever.java

```diff
@@ -129,26 +123,6 @@ public void extendCategoryQueryWith(Consumer<CategoryTreeQuery> categoryQueryHoo
         this.categoryQueryHook = categoryQueryHook;
     }
 
-    /**
-     * Extend the product query part of the category GraphQL query with a partial query provided by a lambda hook that sets additional
-     * fields.
-     *
-     * Example:
-     *
-     * <pre>
-     * {@code
-     * categoryRetriever.extendProductQueryWith(p -> p
-     *     .createdAt()
-     *     .addCustomSimpleField("is_returnable"));
-     * }
-     * </pre>
-     *
-     * @param productQueryHook Lambda that extends the product query
-     */
-    public void extendProductQueryWith(Consumer<ProductInterfaceQuery> productQueryHook) {
```

Comment:
I moved this same functionality into the SearchService, and the extended product query hook is supplied as a parameter in the search method.